### PR TITLE
Added support for EM:Extended test case scenario.

### PR DESCRIPTION
**Pull request by SamirMulani**
1. Added code changes to run the test_idlestate_mode function for a specific period of time as per the time input is given into the .yaml file. 
2. Checking dmesg for errors every 1/4 of the total runtime.
3. Added new .yaml file to get runtime parameter.
Note: The time unit is in second

### CODE REVIEW

**Comment by SamirMulani** [comment]
Run Logs,
```
 avocado run cpupower_monitor.py:CpupowerMonitor.test_dis_ena_idlestate -m cpupower_monitor.py.data/cpupower.yaml --max-parallel-tasks=1
JOB ID     : 471c187c23a00f2313dd6cadbdcd3b718e913c1d
JOB LOG    : /Workspace/avocado-fvt-wrapper/results/job-2023-05-04T05.20-471c187/job.log
 (1/1) cpupower_monitor.py:CpupowerMonitor.test_dis_ena_idlestate;run-3f91: STARTED
 (1/1) cpupower_monitor.py:CpupowerMonitor.test_dis_ena_idlestate;run-3f91: PASS (104.22 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /Workspace/avocado-fvt-wrapper/results/job-2023-05-04T05.20-471c187/results.html
JOB TIME   : 113.00 s
```

File: cpu/cpupower_monitor.py [hunk 1]
```diff
@@ -115,23 +116,36 @@ def test_workload(self):
                           " ebizzy workload")
         self.log.info("cpus have entered idle states after killing work load")
 
-    def test_disable_idlestate(self):
+    def test_dis_ena_idlestate(self):
```
Comment (PraveenPenguin) [inline cpu/cpupower_monitor.py:119]: can you please change this to some meaningful name

File: cpu/cpupower_monitor.py [hunk 2]
```diff
@@ -173,7 +187,8 @@ def test_idle_info(self):
             process.run('ppc64_cpu --smt=%s' % i, shell=True)
             process.run('ppc64_cpu --smt', shell=True)
             output = process.system_output(
-                'cpupower -c %s idle-info | grep offline' % i, shell=True).split()
+                'cpupower -c %s idle-info | grep offline' % i,
+                shell=True).split()
```
Comment (PraveenPenguin) [inline cpu/cpupower_monitor.py:191]: i feel not relevant with change commit , can you please remove this and other places where it has style changes

File: cpu/cpupower.yaml [hunk 3]
```diff
@@ -0,0 +1 @@
+runtime: 15
```
Comment (PraveenPenguin) [inline cpu/cpupower.yaml:1]: what is unit of run time ?

**Review by PraveenPenguin — CHANGES_REQUESTED** [review]

File: cpu/cpupower.yaml (hunk 3 above)
Comment (SamirMulani) [reply cpu/cpupower.yaml:1]: runtime unit is in second.

**Comment by sacsant** [comment]
We should make it mandatory to provide supporting logs which shows that introducing a new option in Yaml does not break existing code. This commit introduced a function name change but failed to update the same in places where the function was called.

**Pull request merged**